etc/logrotate.d/dnf

```diff
@@ -3,5 +3,5 @@
     notifempty
     rotate 4
     weekly
-    create 0600 root root
+    create 0644 root root
```

Comment:
I did a bit of investigation, this is interesting. I believe the `%ghost` [macros we have in the spec](https://github.com/rpm-software-management/dnf/blob/master/dnf.spec#L420) don't create the files ([doc](http://ftp.rpm.org/max-rpm/s1-rpm-inside-files-list-directives.html#S3-RPM-INSIDE-FLIST-GHOST-DIRECTIVE)). Therefore, the `%attr` setting the permissions there AFAICS has no effect and should be removed to prevent the confusion.

The log file is AFAICS created when something is logged for the first time and the permissions are defined by the `umask` of the system.

I don't think it's really necessary to create a new empty file after rotation in the logrotate script, but if we want to do that, the `create` directive seems to supports omitting the `mode` ([doc](https://linux.die.net/man/8/logrotate)), which would allow preserving whatever setting the user put into their `umask`.

Reply:
I see, I have omitted all the settings (mode, owner and group) to preserve the original file's setting.
It seems to work well when tested by running `logrotate --force /etc/logrotate.d/dnf`
Thank you for helping.

Regarding the `%ghost` attributes they do seem to have [some use](https://github.com/rpm-software-management/dnf/pull/1263), so I would keep them.